src/SFA.DAS.EmploymentCheck.Database/Tables/ApprenticeEmploymentCheckMessageQueueHistory.sql

```diff
@@ -0,0 +1,18 @@
+﻿CREATE TABLE [dbo].[ApprenticeEmploymentCheckMessageQueueHistory]
+(
+	[MessageHistoryId] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY,
+    [MessageHistoryCreatedDateTime] DATETIME NOT NULL,
+	[MessageId] UNIQUEIDENTIFIER NOT NULL,
+    [MessageCreatedDateTime] DATETIME NOT NULL,
+    [EmploymentCheckId] BIGINT NOT NULL,
+	[Uln] BIGINT NOT NULL,
+    [NationalInsuranceNumber] VARCHAR(20) NOT NULL,
+    [PayeScheme] VARCHAR(255) NOT NULL,
+    [StartDateTime] DATETIME NOT NULL,
+    [EndDateTime] DATETIME NOT NULL,
+    [EmploymentCheckedDateTime] DATETIME NOT NULL,
+    [IsEmployed] BIT NOT NULL,
```

Comment:
IsEmployed s/b NULLABLE as this table should be an exact copy of delete queue record